pyvo/dal/tap.py

```diff
@@ -138,6 +141,33 @@ def tables(self):
                 vosi.parse_tables(response.raw.read), tables_url)
         return self._tables
 
+    @property
+    def examples(self):
```

Comment:
Sorry to chime in here so late in the game, but the `examples` end point is not part of the standard, is it? If it's not ,shouldn't it be available under a new submodule (`extensions`)? I know of a few other TAP features that are not yet part of the standard but used by at least a couple of service providers but I was reluctant to add them to the core library.

Reply:
Looks like it's in 1.1 but not in 1.0?  (I usually only look at the 1.1) from the new DALI? http://www.ivoa.net/documents/TAP/20190927/REC-TAP-1.1.html section 2

Reply:
Indeed, I must have looked at 1.0. What probably confused me was that I haven't seen the capabilities standard ID for examples (`ivo://ivoa.net/std/DALI#examples`). My read of the standard is that unlike the `/tables` end point, one for `examples` is actually specified in the `capabilities` of the service and could end up being different from `baseURL/examples`